test/extensions/filters/network/common/fuzz/uber_per_readfilter.cc

```diff
@@ -20,14 +20,25 @@ std::vector<absl::string_view> UberFilterFuzzer::filterNames() {
   // Will extend to cover other network filters one by one.
   static std::vector<absl::string_view> filter_names;
   if (filter_names.empty()) {
-    filter_names = {NetworkFilterNames::get().ExtAuthorization,
-                    NetworkFilterNames::get().LocalRateLimit,
-                    NetworkFilterNames::get().RedisProxy,
-                    NetworkFilterNames::get().ClientSslAuth,
-                    NetworkFilterNames::get().Echo,
-                    NetworkFilterNames::get().DirectResponse,
-                    NetworkFilterNames::get().DubboProxy,
-                    NetworkFilterNames::get().SniCluster};
+    const auto factories = Registry::FactoryRegistry<
+        Server::Configuration::NamedNetworkFilterConfigFactory>::factories();
+    const std::vector<absl::string_view> supported_filter_names = {
+        NetworkFilterNames::get().ExtAuthorization,
+        NetworkFilterNames::get().LocalRateLimit,
+        NetworkFilterNames::get().RedisProxy,
+        NetworkFilterNames::get().ClientSslAuth,
+        NetworkFilterNames::get().Echo,
+        NetworkFilterNames::get().DirectResponse,
+        NetworkFilterNames::get().DubboProxy,
+        NetworkFilterNames::get().SniCluster};
+    // Check whether each filter is loaded into Envoy.
```

Comment:
Say why this is necessary:  Some customers build Envoy without some filters.  When they run fuzzing, the use of a filter that does not exist will cause fatal errors.

Reply:
> Say why this is necessary: Some customers build Envoy without some filters. When they run fuzzing, the use of a filter that does not exist will cause fatal errors.

Thanks! Added.